src/common-principles.md

```diff
@@ -653,6 +653,23 @@ for more information.
 
 ## The Inheritance Principle
 
+In some circumstances, there can be multiple data files for which
+all or a subset of the relevant metadata is precisely equivalent.
+Where this occurs,
+it may be preferable to define those metadata *only once*,
+and be placed on the filesystem in such a way that those files
+are deemed to be *applicable* to each relevant data file individually,
+but *not* be erroneously associated with other data files
+to which the metadata contained within are not applicable.
+The Inheritance Principle defines a systematized set of rules by which
```

Comment:
```suggestion
The Inheritance Principle defines a systematized set of rules
```
to make it only 2 `which`, 3 of which cause my mental reset in parsing the sentence.